### DISCUSSION TITLE
Reduce Cost of Dual Spec

### DISCUSSION THREAD

**Discussion by Taznur** [chromiecraft/chromiecraft#6142] [Suggestions]
Hello, I'm a new player to the wow franchise and I just want to say I love the server, especially the new 2x EXP rate, I think it's perfect.

In terms of Dual Spec, it should cost 10g, not 900g for a few important reasons.

- Certain classes like warrior and paladin both have DPS and tank specs, whichever one they go for while levelling doesn't matter BUT at level 40 (when you unlock the choice to buy dual spec), you will have enough talent points to have a decent build. However, the problem is you can't change your build on the fly and this is important in RDFs where you can be either tank or dps like warrior. I'm prot specced warrior and I usually get tank and do fine but sometimes I get DPS and because I don't have dual spec, I can't change to dps which would hinder my performance in the dungeon. For a bit of context, I queued tank and dps and got into the BC dungeon with the nagas and got dps. We were doing the dungeon fine until I got kicked out of the dungeon because I was doing "low dps". One of the party members who voted no to kick me whispered me and explained the situation, if I had dual spec this situation would probably have been avoided. Now you can argue why queue as dps when I'm tank specced and that's because up to this point, it didn't really matter, I would still finish the dungeon no matter what role I was but you can argue that the person who kicked me was toxic (which he was)
- Who has 900g at 40? When I first hit 40, I had around 160g-200g which is no where near the amount I need. There's absolutely no reason for dual spec to be 900g, all it does is improve build availability on the fly when you need to as mentioned above.
- Due to the low pop, dungeon queues can be LOOOOOONG, easily over 30 minutes so having dual spec means you can have a levelling build and dungeon build, meaning you queue for the role that's missing in your group e.g. Fury warrior swaps to prot warrior cuz the group needs a tank.
- I just arrived in outlands and it's awesome but flying mounts is MEGA EXPENSIVE, I think as much as dual spec if I'm not mistaken. If that's the case, when does a person ever have the funds for dual spec? Why is it even in the game when you have to save for the flying mount which is the same price? Why does it exist if players will never be able to buy it while levelling? It is such a useful feature while levelling that in my opinion, should be unlocked for free automatically when you hit 40 or even 30, Whatever you wish the cost to be, it shouldn't be 900g because that is absurd and makes it nearly impossible to buy, especially during crucial times in your levelling journey.

I absolutely love the server, besides the low pop, it has been a great time and I thank you for creating an amazing server. As a person who has never experienced wow, the progressive state of the game makes me feel like I'm playing the game when it first came out with added QoL. You guys are doing a great job and I hope for a bright future ahead.

**Comment by heyitsbench** [comment]
I would like to point out that the normal price of dual specialization is one-thousand gold, and is only reduced due to the lower level cap iirc. It was also a hundred gold cheaper at level cap of 60.

**Comment by Contrubius** [comment]
What do we do with people who already purchased dual spec for dozens of characters for the respectable price of 800 or 900 gold? This is the exact point why respec price is free, so you can test and experiment with talents and gear. I consider it a giant slap in the face to all the people who invested thousands of gold on max level for the service , and it is not like certain talents make a huge difference on low level anyway.

In most cases, because of the content drought, there is absolutely nothing to do but grind anyway, just like the original TBC, so it is not like the desired sum of around 1700 gold cannot be achieved in relatively short period of time. And free respec from your trainer is one hearthstone away.

**Comment by Taznur** [comment]
*" What do we do with people who already purchased dual spec for dozens of
characters for the respectable price of 800 or 900 gold?"*
You can just refund them partly, for example if you decide to change the
cost to 50g, then you would refund the player who already has dual spec
750g or 850g.

"This is the exact point why respec price is free, so you can test and
experiment with talents and gear."
If respec price asn't free then it would just be another complaint of mine,
you're not blizzard and you don't have to follow their principles to an
absolute tea. They make great games but some features are EXTREMELY
outdated. For example, Blade & Soul lets respec your talent for FREE and
ANYWHERE, imagine how fun and free it would be if WoW was like this?

*"and it is not like certain talents make a huge difference on low level
anyway."*
It doesn't but this contradicts you're earlier point of why respec is free
because "you can test and experiment with talents and gear". Trying new
builds should be encouraged during all points of your WoW journey and it
just so happens that levelling is the longest and arguably the most fun
part. My earlier point stands in my post in github where some classes would
really benefit from dual spec, like a tank warrior switching to fury
because the RDF gave them DPS.

"so it is not like the desired sum of around 1700 gold cannot be achieved
in relatively short period of time."
The grind is fine but why so much for dual spec? It makes no sense
especially when it's unlocked at 40, where you're no where near the amount
needed. Like I said in my github post, there's no reason for the feature to
exist if majority are not going to be able to access it, especially during
times where it would be really useful like a mentioned above. Flying mounts
are already 900g, why make the grind even more unnecessarily long? Like I
said before, you don't have to adopt Blizzards outdated principles into
this private server, that's the whole point, to change certain features to
make it a better game overall.

"And free respec from your trainer is one hearthstone away."
1 hearthstone away which is on a 30 minute cooldown. But seriously,
majority of players are going to be out and about questing or grinding
professions which is the whole point of the game, not camp your class
trainer in case you need a certain build for a certain situation. Going to
any major city with your class trainer takes a considerable amount of time
for such a little but important thing, it takes away from the experience of
playing WoW and it just makes you think "Damn I need to go respec". And
let's be honest, flights take way too long, sure they're considerably
faster than walking but this is another one of Blizzards outdated feature
that majority of other mmos don't implement for a reason, it takes too
long. So long in fact that I can watch a youtube video in between long
flights but this is a topic for another day.

I appreciate your time to reply to my post.

**Comment by Taznur** [comment]
Hello, it's been a few days and I'm wondering if you got my reply, I would
like to know your thoughts.

**Comment by Squirrelznut** [comment]
No one has ever been refunded stuff when things got cheaper at a later date. 
That just never happened in WoW so why should it happen on Chromie?
Should it be cheaper? I'm certainly not against it, but I'd rather people be wealthier (see not cutting gold form quests at 70)

**Comment by Amrx2** [comment]
Hi there, currently Dual-spec is discounted from its original price in 3.3.5a of 1000 gold, to 900 gold. This has been slowly increased in each level bracket from early vanilla to reflect the increased ease in farming gold. The cost of 10 gold is a change brought in Cataclysm.

Dual-spec as well as Flying is one of two goldsinks we currently have in the game that serves a vital purpose of dampening total gold numbers in game. To assist with players respec costs are free until they can farm the gold needed to purchase this. Thank you for the suggestion but this isn't one we are going to change.

Cheers,
Construct.
